pkg/controller/kameletbinding/error_handler.go

```diff
@@ -46,11 +46,7 @@ func maybeErrorHandler(errHandlConf *v1alpha1.ErrorHandlerSpec, bindingContext b
 			}
 		}
 
-		err = setErrorHandlerConfiguration(errorHandlerBinding, errorHandlerSpec)
-		if err != nil {
-			return nil, errors.Wrap(err, "could not set integration error handler")
-		}
-
+		setErrorHandlerConfiguration(errorHandlerBinding, errorHandlerSpec)
```

Comment:
Should we really remove this error handling here?  I see in this func it's not needed as it's already validated above, but what if some one else found `setErrorHandlerConfiguration()` useful and wanted to invoke it somewhere else?
I feel it's safe to keep this as is along with the `setErrorHandlerConfiguration()` func.